api/src/api/competition_alpha/competition_schema.py

```diff
@@ -42,7 +45,10 @@ class CompetitionAlphaSchema(Schema):
         metadata={"description": "The opportunity ID that the competition is associated with"}
     )
 
-    competition_forms = fields.List(fields.Nested(CompetitionFormAlphaSchema()))
+    competition_forms = fields.List(
+        fields.Nested(CompetitionFormAlphaSchema()),
+        metadata={"description": "List of forms required for this competition", "example": []},
```

Comment:
Leave the example out for an array and it'll just default to the example being an array with one of the competition form object.
```suggestion
        metadata={"description": "List of forms required for this competition"},
```